src/main/java/com/google/sps/servlets/AnalyzedThread.java

```diff
@@ -0,0 +1,84 @@
+package com.google.sps.servlets;
+
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.util.Random;
+
+public class AnalyzedThread {
```

Comment:
nit: Can you add a small Javadoc line to explain what the class is doing? For example:

```java
/** Represents a thread that's been analyzed by the NLP API. */